src/containers/MenuContainer.js

```diff
@@ -0,0 +1,17 @@
+import Menu from '../components/Menu.js';
+import { connect } from 'react-redux';
+import actions from "../actions";
+
+const mapStateToProps = (state) => {
+  return {}; // TODO - complete
+};
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    handleEvent: (event) => {
```

Comment:
We have a number of buttons, this is meant to correctly handle which ever one we click on. I've added a switch case to do so :)